tools/labs/qemu/kernel_config.x86

```diff
@@ -4,28 +4,27 @@
 CONFIG_BLK_DEV_INITRD=y
 # CONFIG_COMPAT_BRK is not set
 CONFIG_SLAB=y
-# CONFIG_64BIT is not set
```

Comment:
@dbaluta Can you please check that we are still building 32bits?

Reply:
It looks like 32bits to me. 

```
file drivers/base/power/clock_ops.o
drivers/base/power/clock_ops.o: ELF 32-bit LSB relocatable, Intel 80386, version 1 (SYSV), with debug_info, not stripped
```